app/models/concerns/common_metadata.rb

```diff
@@ -19,6 +19,9 @@ module CommonMetadata
     property :state, predicate: ::F3Access.objState, multiple: false do |index|
       index.as :stored_searchable, :facetable
     end
+    property :workflow_note, predicate: ::RDF::Vocab::MADS.note do |index|
```

Comment:
Jeez...[everyone's defined a note](http://lov.okfn.org/dataset/lov/terms?q=note) and none seem right...